src/components/side-navigation/side-navigation-item.component.ts

```diff
@@ -0,0 +1,87 @@
+import { Component, Input, ViewEncapsulation, ContentChildren, QueryList, Renderer2, ElementRef, Optional, SkipSelf } from '@angular/core';
+import { SideNavigationService } from './side-navigation.service';
+
+@Component({
+  selector: 'ux-side-navigation-item',
+  templateUrl: './side-navigation-item.component.html',
+  host: {
+    '[class.active]': 'active',
```

Comment:
If I understand correctly, this needs to change. We need to distinguish between the node which is active (i.e. the current navigation route) and ancestors of the node which is active. As I said before, only the active node will be highlighted in the MF theme, so this is important to consider.